### PR TITLE
en: add doc for tidbmonitor shards

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added, or deleted? (Required)

Add a doc for enabling TidbMonitor shards.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1415
- Other reference link(s):

### CODE REVIEW

File: en/enable-monitor-shards.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+summary: Learn how to use shards for TidbMonitor.
+---
+
+# Enable Shards for TidbMonitor
+
+This document describes how to use shards for TidbMonitor.
+
+## Shards
+
+TidbMonitor collects monitoring data for a single TiDB cluster or multiple TiDB clusters. When the amount of monitoring data is large, the computing power of one TidbMonitor might hit a bottleneck. In this case, it is recommended to use shards of Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/). This feature performs `hashmod` on `__address__` to divide the monitoring data of multipel targets (`Targets`) into multiple TidbMonitor Pods.
```
Comment (qiancai) [inline en/enable-monitor-shards.md:12]:
```suggestion
TidbMonitor collects monitoring data for a single TiDB cluster or multiple TiDB clusters. When the amount of monitoring data is large, the computing capacity of one TidbMonitor might hit a bottleneck. In this case, it is recommended to use shards of Prometheus [Modulus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/). This feature performs `hashmod` on `__address__` to divide the monitoring data of multiple targets (`Targets`) into multiple TidbMonitor Pods.
```

**Review by qiancai — COMMENTED** [review]
@ran-huang I've completed the review and made one comment. PTAL. Thanks.

**Review by qiancai — APPROVED** [review]
LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**